src/main/java/io/codelex/arrays/exercises/Exercise_7.java

```diff
@@ -0,0 +1,90 @@
+package io.codelex.arrays.exercises;
+
+import java.util.Scanner;
+
+public class Exercise_7 {
+    public static void main(String[] args) {
+        ticTacToe();
+    }
+
+    private static void ticTacToe() {
+        Scanner scan = new Scanner(System.in);
+        char[][] gameBoardArray = new char[3][3];
+        for (int i = 0; i < 3; i++) {
+            for (int j = 0; j < 3; j++) {
+                gameBoardArray[i][j] = '-';
+            }
+        }
+        String gameBoardString = printBoard(gameBoardArray);
+        System.out.println(gameBoardString);
+
+        char xOrO = 'X';
+        boolean isFirstPlayer = true;
+        boolean isWinner = false;
```

Comment:
`The default value of any Object , such as Boolean , is null . The default value for a boolean is false.`

šeit arī vari izlaist piešķirt vērtību

Reply:
Ja isWinner nepiešķir vērtību 66. rindiņa ir neapmierināta - Variable 'isWinner' might not have been initialized.
Ir kāds cits veids, lai viņu nomierinātu?